src/domains/idpay-app/10_p7m_certificate.tf

```diff
@@ -32,7 +32,7 @@ resource "azurerm_key_vault_secret" "ranking_p7m_private_key" {
   name            = "ranking-p7m-key"
   value           = tls_private_key.p7m_key[0].private_key_pem
   content_type    = "text/plain"
-  expiration_date = tls_self_signed_cert.p7m_self_signed_cert[0].validity_end_time
+  expiration_date = replace(formatdate("YYYY-MM-DD'T'hh:mm:ssZ", timeadd(tls_self_signed_cert.p7m_self_signed_cert[0].validity_end_time, "-59m59s500ms")), "+01:00", "Z")
```

Comment:
@lucaconsalvi smart trick! But maybe we can just (and safely) ignore the proposed change.

Reply:
😁 I had thought that but I didn't know if ignore it was ok for us, anyway I'll procede in this way thx!